store.go

```diff
@@ -1175,7 +1178,19 @@ func (s *store) CreateContainer(id string, names []string, image, layer, metadat
 			},
 		}
 	}
-	clayer, err := rlstore.Create(layer, imageTopLayer, nil, "", nil, layerOptions, true)
+	if options.Flags == nil {
+		options.Flags = make(map[string]interface{})
+	}
+	if options.Flags["ProcessLabel"].(string) == "" {
```

Comment:
What happens, or should happen, if the "MountLabel" flag is set but the "ProcessLabel" flag is not?  If this type assertion fails, what happens?

Reply:
I changed it to thrown an error.